tests/properties.hs

```diff
@@ -32,7 +32,7 @@ import Test.QuickCheck
 import Test.Framework
 import Test.Framework.Providers.QuickCheck2
 import Data.Char (isAlphaNum, isAscii, toUpper)
-import Data.Text.Strict.Lens
+import qualified Data.Text.Strict.Lens as Text
```

Comment:
This is a sign we might want a type-class. If we want these to begin with, it is.

Reply:
Good and interesting point!

Reply:
If we did offer a type class for this, there's a question of what we should do with the existing `prefixed` and `suffixed` functions in `Data.List.Lens`. Would it be preferable to remove them in favor of a type class that provides a single API entrypoint for all types, or would we want to offer the type class alongside specialized `prefixed`/`suffixed` functions in `Data.List.Lens`, `Data.Text.Strict.Lens`, etc.?

Reply:
only type-class seems sane. It could be re-exported by `Data.List.Lens` etc (c.f Cons).